### PR TITLE
Health Dashboard setup for QA

### PR DESCRIPTION
**Pull request by neel-tw**

### CODE REVIEW

File: data/default/ACCESSCONTROL-ROLES/roles.json [hunk 1]
```diff
@@ -21,6 +21,21 @@
       "code": "SUPERUSER",
       "name": "Super User",
       "description": "Super User. Can change all master data and has access to all the system screens."
+    },
+    {
+      "code": "NATIONAL_SUPERVISOR",
```
Comment (egov-joy) [inline data/default/ACCESSCONTROL-ROLES/roles.json:26]: As discussed separately, having separate roles per boundary level is not in line with the required flexibility on platform. Please advise if product and platform team have agreed to this approach

Comment (neel-tw) [reply data/default/ACCESSCONTROL-ROLES/roles.json:26]: Have got confirmation on the above from Abhishek & Mrunal. Mrunal has also discussed this with Ganshyam and got confirmation on Jan, 11.

Comment (egov-joy) [reply data/default/ACCESSCONTROL-ROLES/roles.json:26]: @GhanshyamRawat-eGov please confirm that we need these additional boundary level specific roles.

Comment (neel-tw) [reply data/default/ACCESSCONTROL-ROLES/roles.json:26]: Hi @GhanshyamRawat-eGov, Could you please confirm on the above

Comment (GhanshyamRawat92) [reply data/default/ACCESSCONTROL-ROLES/roles.json:26]: @egov-joy They have that requirment that dashboards are going to be boundary specific and that is why they are creating separate dashboards at each level.

**Review by GhanshyamRawat92 — APPROVED** [review]

**Review by shashwat-egov — APPROVED** [review]

**Pull request merged**